### PR TITLE
streams.part1.exercise2 (Turenko)

### PR DESCRIPTION
**Pull request by turenkoaa**

### CODE REVIEW

File: src/test/java/streams/part1/exercise/Exercise2.java [hunk 1]
```diff
@@ -87,7 +85,7 @@ public void combineMapsAndPutToLatest() {
         newValues.put(alex, Status.DECLINED);
         newValues.put(helen, Status.PENDING);
 
-        // TODO реализация
+        oldValues.forEach((k, v) -> newValues.merge(k, v, (oldValue, newValue) -> oldValue));
```
Comment (AleksandrRadchenko) [inline src/test/java/streams/part1/exercise/Exercise2.java:88]: Seems like the task was to use "latest" enum, according to their provided sorting. It means using enum.ordinal()

**Review by AleksandrRadchenko — CHANGES_REQUESTED** [review]
Please, make base branch elefus:turenko

File: src/test/java/streams/part1/exercise/Exercise2.java (hunk 1 above)
Comment (Onoxider) [reply src/test/java/streams/part1/exercise/Exercise2.java:88]: The word latest(should be latter actually, like "former and latter") refers to the latter map i.e. newValues, therefore the solution is correct

**Review by Onoxider — APPROVED** [review]

**Review by elefus — CHANGES_REQUESTED** [review]
Исправить целевую ветку на turenko

**Comment by turenkoaa** [comment]
This branch should be merge, I think.

**Comment by turenkoaa** [comment]
This branch should be merged, I think.

**Review by elefus — APPROVED** [review]

**Pull request merged**